### DISCUSSION TITLE
Home.md light / dark style.

### DISCUSSION THREAD

**Discussion by Karlstens** [vuejs/vitepress#3315] [Q&A] [answered]
I'm attempting to change the colors used with the default homepage, but have them influenced by the dark / light toggle. Unfortunately the Theme Home Page documentation doesn't cover specifically toggling <style> to work with dark/light theme.

https://vitepress.dev/reference/default-theme-home-page

Within my home page Markdown content (underneath the home.md frontmatter), I have this style for the fancy blur graphic background. I'd like the colors to differ between light / dark toggle. Currently, the colors show the same for both light/dark theme.

```md
<style>
:root {
  --vp-home-hero-name-color: transparent;
  --vp-home-hero-name-background: -webkit-linear-gradient(120deg, #41d1ff 30%, #41d1ff);

  --vp-home-hero-image-background-image: linear-gradient(-45deg, #000000 50%, #47caff 50%);
  --vp-home-hero-image-filter: blur(44px);
}

@media (min-width: 640px) {
  :root {
    --vp-home-hero-image-filter: blur(56px);
  }
}

@media (min-width: 960px) {
  :root {
    --vp-home-hero-image-filter: blur(68px);
  }
}
</style>
```

Keen to understand how I may be able to achieve this.

**Comment by sharpchen** [comment, marked as answer]
May be `.dark` would work? That's how global style work for adaptive theme. https://github.com/vuejs/vitepress/blob/e4bf1e48e6a1592d583b218425c1fa7497955dc5/src/client/theme-default/styles/vars.css#L13
```css
.dark {
  /* ... */
}
```

**Reply by Karlstens** [reply]
Ahh perfect. I understand now, thank you! 🌘
